### PR TITLE
Multiple articles: Re-work article pivots

### PR DESCRIPTION
**Pull request by JavierMatosD**
- Changes article pivots from OS selection to shell selection
- Adds a screenshot for how to pull up the VS developer powershell terminal
- Crops the env variable screenshot in the VS article to be usable for other articles
- Fixes syntax for various terminal options
- Adds another pivot group, `os-runner`,  with the following pivots: `windows-runner`, `linux-runner`
- Updates the binary caching articles to use new pivot groups.
- Updates the get-started articles to use new pivot groups.
- Adds `cmd` instructions for the get started with CMake / VS article

### CODE REVIEW

**Review by ras0219-msft — COMMENTED** [review]
Two more suggestions on the images:
1. Crop the Developer PowerShell screenshot to focus on the menuing without the solution explorer / find symbol results / etc.
2. In general, increase the zoom level / reduce the resolution. The review site compresses the images to the point that they're currently _almost_ illegible. Perhaps try resizing the visual studio window to be as small as possible before taking the screenshot? Then hopefully the details should survive size-based compression.

File: vcpkg/get_started/get-started.md [hunk 1]
```diff
@@ -59,6 +59,17 @@ This tutorial shows you how to create a C++ "Hello World" program that uses the
...
+    ::: zone-end
+    ::: zone pivot="shell-powershell"
+
+    ```powershell
+    $env:VCPKG_ROOT = "C:\path\to\vcpkg"
+    $env:PATH = "$env:VCPKG_ROOT;$env:PATH"
+    ```
+
+    > [!NOTE]
+    > Setting the `VCPKG_ROOT` environment variable in this way only affects the current shell session. To make this change permanent across sessions, you would typically modify them through the Windows System Properties.
```
Comment (data-queue) [inline vcpkg/get_started/get-started.md:71]:
```suggestion
    > Setting the `VCPKG_ROOT` environment variable in this way only affects the current shell session. To persist this change across sessions, you would typically modify them through the Windows System Properties.
```

Comment (data-queue) [inline vcpkg/get_started/get-started.md:71]: I couldn't find what the Windows System Properties were and how to set them. Can you explain what that is?

File: vcpkg/get_started/get-started-vs.md [hunk 2]
```diff
@@ -41,9 +41,15 @@ This tutorial shows you how to create a C++ "Hello World" program that uses the
...
+    Open the built-in Developer PowerShell window in Visual Studio.
+    
+    :::image type="complex" source="../resources/get_started/visual-studio-developer-powershell.png" alt-text="opening built-in developer powershell":::
+        Screenshot of Visual Studio UI for the built-in PowerShell developer window
+    :::image-end:::
+
+    Run the following commands:
 
-    ```PowerShell
+    ```powershell
```
[the remark this answers is not in the document]
Comment (JavierMatosD) [reply vcpkg/get_started/get-started-vs.md:52]: This is true, but we explicitly tell the user to use VS's built-in developer powershell. I could provide instructions with the built-in developer command prompt as well.

File: vcpkg/consume/binary-caching-local.md [hunk 3]
```diff
@@ -42,15 +42,15 @@ access and a second one with only read access.
 
 Create the two binary cache locations (substitute with any locations of your choosing):
 
-::: zone pivot="os-windows"
+::: zone pivot="shell-powershell, shell-cmd"
 
 ```PowerShell
 mkdir D:\vcpkg\binary-cache
```
[the remark this answers is not in the document]
Comment (JavierMatosD) [reply vcpkg/consume/binary-caching-local.md:48]: I will switch to `console` since the instructions are the same for cmd and PowerShell in this case.

**Review by AugP — APPROVED** [review]

**Pull request merged**